.github/workflows/testing.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
     services:
       postgres:
-        image: postgres
+        image: postgis/postgis
         ports: ["5432:5432"]
         env:
           POSTGRES_PASSWORD: postgres
```

Comment:
Previously we had to include the following step after ruby's setup. Not sure if is still needed, good to check 

```
- name: Enable postgis extension
  uses: huaxk/postgis-action@v1
  with:
    postgresql version: '11'
```
cc https://github.com/unboxed/bops/pull/8

Reply:
Hey thanks.
CI seems to be running ok though so maybe not needed

Reply:
All the `huaxk/postgis-action` seems to do is call docker-run with the `postgis/postgis` image. I'm not sure that we'd get anything from it that setting the image directly doesn't do.